icekit/bin/manage.py

```diff
@@ -7,10 +7,15 @@
 
 
 if __name__ == "__main__":
-    if not os.environ.get('ICEKIT_PROJECT_DIR'):
+    ICEKIT_PROJECT_DIR = os.environ.get('ICEKIT_PROJECT_DIR')
+    if not ICEKIT_PROJECT_DIR:
         print("'ICEKIT_PROJECT_DIR' is not defined.", file=sys.stderr)
         exit(1)
 
+    # Add project dir to Python path
+    if ICEKIT_PROJECT_DIR not in sys.path:
+        sys.path.insert(0, ICEKIT_PROJECT_DIR)
```

Comment:
It only seems to be necessary in Django's `manage.py`, python in general can already load items from the project dir. I made this change because at the time I was trying to load a custom named settings module for the project instead of `icekit_settings`, and without adding the dir to python's path Django couldn't find it. I'm not sure why...